code/__DEFINES/dcs/signals/signals_mob/signals_mob_living.dm

```diff
@@ -1,2 +1,5 @@
 ///from base of mob/living/death(): (gibbed)
 #define COMSIG_LIVING_DEATH "living_death"
+
+/// used in mob/living/say to modify variables such as font_size and speech_sound
+#define COMSIG_LIVING_SAY "living_say"
```

Comment:
Use/port `COMSIG_MOB_SAY` so it's compatible with TG for future use, instead of this signal

Reply:
![image](https://github.com/Aurorastation/Aurora.3/assets/22774890/53431c30-ee2d-46a3-92a3-dbfa2c2dd444)

from what i can see the TG way of handling it is different, cuz they pass the args instead of these specific vars (obtained from handle_speech_sound())

if someone wants to port TG's stuff, they can just replace this signal

Reply:
Yes, they would be in the last parameter of the args, called `message_mods`

We don't need a snowflake signal to only handle this when we already have a reference with a signal that can handle this use case just aswell, even less we need to use strings to get the arguments around, the keys of the list emitted in the signal should be using defines, so either way this isn't correct